proxy/proxy.go

```diff
@@ -13,6 +12,8 @@ import (
 	"strconv"
 	"strings"
 
+	"github.com/leijurv/gb/compression"
```

Comment:
useless newlines between these imports

Reply:
`goimports` does this automatically :^)

Reply:
i just tested it and it does not do that